library/std/src/os/unix/net/datagram.rs

```diff
@@ -883,6 +883,25 @@ impl UnixDatagram {
         self.0.passcred()
     }
 
+    /// Set the id of the socket for network filtering purpose
+    /// and is only a setter.
```

Comment:
It's not clear to me what useful information _"and is only a setter"_ is meant to convey here. Can this sentence be just the following?&mdash;

> Set the id of the socket for network filtering purposes.